tool.sh

```diff
@@ -46,6 +47,8 @@ elif [ "$1" == "prod" ]; then
   readme
   # build and upload package to prod pypi
   python setup.py sdist upload -r pypi
+elif [ "$1" == "update" ]; then
+  git submodule foreach git pull origin master
```

Comment:
I think so. If we use other submodules I don't see how will we hurt from newest versions. 

But what do you have in mind?